@navikt/core/css/layout-container.css

```diff
@@ -0,0 +1,28 @@
+.navds-layout-container {
+  padding-left: var(--ac-l-container-padding-xs, var(--navds-spacing-4));
```

Comment:
Hva er motivasjonen for å gå for en kortere prefix på CSS-variabler?

Reply:
Det lurte jeg også på. Mistenker at det er inspirert av Shopify 😊

Reply:
Det + raskere å skrive. Tokens blir ganske sparsomt brukt i teamene i dag, så håper dette kan hjelpe med bruk. Med nye fargetokens blir eks knappefarge `--a-surface-action` vs `--navds-semantic-color-interaction-primary`.
Når vi gjør token oppdateringen i v2 blir alle semantiske farge-tokens oppdaterte og tenker vi kan "rebrande" til `--a` i samme slengen.

Reply:
Cool! Jeg har tro at det er et godt grep. 

Bare vær litt forsiktig med å lage altfor kryptiske navn på ting. `--ac-l-` trekker den strikken litt langt, synes jeg.